src/components/ControlPanel/ControlPanel.module.css

```diff
@@ -0,0 +1,182 @@
+  font-size: 14px;
+  font-weight: 300;
+  border-color: var(--color-btn-border);
+  color: var(--color-btn-text);
+  background-color: var(--color-btn-bg);
+  transition: background-color .3s ease-in-out, border-color .3s ease-in-out;
+}
+
+.searchInput::-webkit-search-cancel-button {
+  -webkit-appearance: none;
+  appearance: none;
+}
+
+.searchInput:focus {
+  outline: none;
+  color: var(--white);
+  background-color: var(--color-btn-bg-hover);
+  border: 1px solid var(--color-btn-bg-hover);
+}
+
+.searchBtn {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: 40px;
+  width: 40px;
+  padding: 10px;
+  border-radius: 50px;
+  border: none;
+  background-color: var(--red-main);
+  transition: background-color .3s ease-in-out;
+}
+
+.searchBtn:focus {
+  outline: none;
+  background-color: var(--red-hover);
+}
+
+.searchIcon {
+  height: 100%;
+  width: auto;
+  fill: none;
+}
+
+.filterBar {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: 15px;
+  row-gap: 10px;
+}
+
+.filterBtn {
+  cursor: pointer;
+  padding: 10px 20px;
+  border-radius: 50px;
+  font-family: Montserrat, sans-serif;
+  font-size: 14px;
+  font-weight: 600;
+  border: 1px solid;
+  border-color: var(--color-btn-border);
+  color: var(--color-btn-text);
+  background-color: var(--color-btn-bg);
+  transition: background-color .3s ease-in-out,
+    border-color .3s ease-in-out,
+    color .3s ease-in-out;
+}
+
+.filterBtn:focus {
+  outline: none;
+  color: var(--white);
+  border-color: var(--red-main);
+  background-color: var(--red-main);
+}
+
+.sortBar {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  column-gap: 10px;
+}
+
+.sortLabel {
+  color: var(--color-text);
+  font-size: 14px;
+  font-weight: 600;
+}
+
+.sortSelect {
+  position: relative;
+  cursor: pointer;
+  appearance: none;
+  padding: 10px 20px;
+  font-family: Montserrat;
+  font-size: 14px;
+  font-weight: 600;
+  border-radius: 50px;
+  border: 1px solid;
+  border-color: var(--color-btn-border);
+  color: var(--color-btn-text);
+  background-color: var(--color-btn-bg);
+  transition: background-color .3s ease-in-out,
+    border-color .3s ease-in-out,
+    color .3s ease-in-out
+}
+
+.sortSelect:focus {
+  outline: none;
+  color: var(--white);
+  border-color: var(--red-main);
+  background-color: var(--red-main);
+}
+
+@media (any-hover: hover) {
+
+  .searchInput:hover {
+    outline: none;
+    background-color: var(--color-btn-bg-hover);
+    border: 1px solid var(--color-btn-bg-hover);
+  }
+
+  .searchBtn:hover {
+    background-color: var(--red-hover);
+  }
+
+  .filterBtn:hover {
+    color: var(--white);
+    border-color: var(--red-main);
+    background-color: var(--red-main);
+  }
+
+  .sortSelect:hover {
+    color: var(--white);
+    border-color: var(--red-main);
+    background-color: var(--red-main);
+  }
+}
+
+@media (width >=769px) {
```

Comment:
Please correct to the design 
![image](https://github.com/snischuk/fe-react-2024/assets/98031768/e60a3cb4-72e4-47c0-864a-52ddb4e87a9b)